services/src/main/java/org/keycloak/services/resources/LoginActionsService.java

```diff
@@ -1050,7 +1052,10 @@ public void ignore() {
         } else if (context.getStatus() == RequiredActionContext.Status.FAILURE) {
             response = interruptionResponse(context, authSession, action, Error.CONSENT_DENIED);
         } else {
-            throw new RuntimeException("Unreachable");
+            if (!Objects.equals(factory.getId(), DeleteAccount.PROVIDER_ID)) {
+                throw new RuntimeException("Unreachable");
+            }
+            response = Response.status(302).location(Urls.accountPage(session.getContext().getUri().getBaseUri(), realm.getName())).build();
```

Comment:
Remove this - it needs to be handled in DeleteAction, not LoginActionsService

Reply:
gotcha